week2/problem2.6.1.1.go

```diff
@@ -0,0 +1,16 @@
+package main
+
+import "fmt"
+
+//Given a string, return a string where for every char in the original, there are two chars.
+
+func main() {
+
+	givenString := "tarasnikita"
+	var doubleCharString string
+	for i, _ := range givenString {
+		doubleCharString += string(givenString[i]) + string(givenString[i])
```

Comment:
for _, c := range givenString {
		doubleChar := string(c) + string(c)
		doubleCharString += doubleChar

	}